.drone.yml

```diff
@@ -11,11 +11,10 @@ matrix:
 
 pipeline:
   fmt_and_lint:
-    image: python:3.8-alpine
+    image: nixos/nix:2.3.6
```

Comment:
i feel this over complicates the build process as well as gives the assumption that you need nix to run these tests.

Reply:
I can make clear that we don't require nix to run the tests? It does neatly pin the black/pylama(and co) versions which is nice for CI.